frontend/awx/access/credentials/hooks/useCredentialsValidate.tsx

```diff
@@ -0,0 +1,54 @@
+import { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { PromptFormValues } from '../../../resources/templates/WorkflowVisualizer/types';
+import { Credential } from '../../../interfaces/Credential';
+import { requestGet } from '../../../../common/crud/Data';
+import { awxAPI } from '../../../common/api/awx-utils';
+
+export function useCredentialsValidate() {
+  const { t } = useTranslation();
+  return useCallback(
+    async (selectedCredentials: PromptFormValues['credentials']) => {
+      const originalCredentials: number[] = [];
+      const newCredential: Credential[] = [];
```

Comment:
```suggestion
      const newCredentials: Credential[] = [];
```
We could rename `newCredential` to `newCredentials` for consistency